tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -303,17 +309,10 @@ describe(__filename, () => {
     ).toHaveAttribute('href', url);
   });
 
-  it('passes disabled to AMInstallButton based on what is returned from _getClientCompatibility', () => {
-    const addon = fakeAddon;
-
-    const _getClientCompatibility = jest.fn().mockReturnValue({
-      compatible: true,
-    });
-
+  it('passes disabled to AMInstallButton based on what is returned from _getClientCompatibility', async () => {
     render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
       version: createInternalVersionWithLang(fakeVersion),
+      addonManagerOverrides: { hasAddonManager: false },
```

Comment:
It's not clear why this is needed now. Did `hasAddonManager` default to `false` before?